Resources/Textures/Clothing/Uniforms/Jumpskirt/qmturtleskirt.rsi/meta.json

```diff
@@ -1,7 +1,7 @@
 {
   "version": 1,
   "license": "CC-BY-SA-3.0",
-  "copyright": "Derived from beestation qmturtleneck sprite from beestation at commit https://github.com/BeeStation/BeeStation-Hornet/commit/a11e7468ca552cfc97c9164c69ccdafff09e794c and modified by Emisse for Space Station 14",
+  "copyright": "Colour insperation from Wizden's QM Turtle, Sprites by Github KittenColony / Discord kittencolony (297865728374210561)",
```

Comment:
Just checking that you erased all of the wizden sprites if you are doing this?

Reply:
QM turtleneck uses my own top template. The others were hueshifts and it feels wrong to change the copyright after just that